logging/config/cluster-logging-management.adoc

```diff
@@ -10,9 +10,9 @@ In order to modify certain components managed by the Cluster Logging Operator or
 
 In unmanaged state, the operators do not respond to changes in the CRs. The administrator assumes full control of individual component configurations and upgrades when in unmanaged state.
 
-In managed state, the Cluster Logging Operator (CLO) responds to changes in the Cluster Logging Custom Resource (CR) and attempts to update the cluster to match the CR.
+In managed state, the Cluster Logging Operator (CLO) responds to changes in the Cluster Logging Custom Resource (CR) and attempts to update the {product-title} cluster to match the CR.
```

Comment:
@mburke5678 what does `{product-title}` mean? Is it equal to `OpenShift`? If yes, I think here should be something like `logging cluster` or `clusterlogging`.

Reply:
The {product-title} is a variable that places the OpenShift project in the docs (OpenShift Container Platform, OpenShift Dedicated, OpenShift Online, OKD, and so forth).

Reply:
I changed this to _adjusts the logging deployment accordingly._ to match the wording we use on the **Understanding cluster logging** page. [1]
`
In managed state, the Cluster Logging Operator (CLO) responds to changes in the Cluster Logging Custom Resource (CR) and adjusts the logging deployment accordingly.`

[1] https://docs.openshift.com/container-platform/4.3/logging/cluster-logging.html#cluster-logging-about_cluster-logging